docs/api.md

```diff
@@ -2191,6 +2192,17 @@ The method evaluates the XPath expression relative to the elementHandle. If ther
 
 This method returns the bounding box of the element (relative to the main frame), or `null` if the element is not visible.
 
+#### elementHandle.boxModel()
+- returns: <[Promise]<?[Object]>>
+  - content <[Array]<[number]>> Content box, represented as a [quad](https://chromedevtools.github.io/devtools-protocol/tot/DOM#type-Quad).
```

Comment:
Let's have an array of objects with`{x, y}` - that will be a more friendly format to work with.

Reply:
done